src/expr/src/explain/json.rs

```diff
@@ -43,7 +43,7 @@ where
         let sources = self
             .sources
             .iter()
-            .map(|ExplainMultiPlanSource { id, op, .. }| {
+            .map(|ExplainSource { id, op, .. }| {
                 serde_json::json!({
                     "id": id,
                     "op": op
```

Comment:
Noted! Is the expectation that JSON should generally have everything the text output does, just in a more machine-readable form?

Reply:
Yes, so that we can:

1. Validate properties of physical plans in tests with `jq` without printing them out.
2. Expose plans for other types of interpretation (think about a better plan visualization for your materialized views in the web console).